examples/image_processing/inverse_deconv.cpp

```diff
@@ -0,0 +1,63 @@
+/*******************************************************
+ * Copyright (c) 2014, ArrayFire
+ * All rights reserved.
+ *
+ * This file is distributed under 3-clause BSD license.
+ * The complete license agreement can be obtained at:
+ * http://arrayfire.com/licenses/BSD-3-Clause
+ ********************************************************/
+
+#include <stdio.h>
+#include <arrayfire.h>
+#include <af/util.h>
+#include <cstdlib>
+
+using namespace af;
+
+const unsigned ITERATIONS = 96;
+const float RELAXATION_FACTOR = 0.05f;
+
+array normalize(const array &in)
+{
+    float mx = max<float>(in.as(f32));
+    float mn = min<float>(in.as(f32));
+    return (in-mn)/(mx-mn);
+}
+
+int main(int argc, char* argv[])
+{
+    int device = argc > 1 ? atoi(argv[1]) : 0;
```

Comment:
Simple and better way is to merge inverse and iterative deconvolution examples. I didn't do that because CPU backend would take too much to execute.

It is not calling just one function, it displays(should, unless it was removed when I weeded out Wiener filter) three difference images -  normal input, smoothed/blurred input, deconvolved output.